index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
+                                        <ul class="first-list">
+                                            <li>CANOPY</li>
+                                            <li>Back End Dev</li>
+                                            <li>2015</li>
+                                        </ul>
+                                        <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                                        <ul class="blue-list">
+                                            <li>HTML</li>
+                                            <li>Css</li>
+                                            <li>javascript</li>
+                                        </ul>
+                                        <button class="work-btn btn">see project</button>
+                                    </div>
+                                </section>
+            </section>
+            <!--the about myself section--> 
+            <section class="about">
+                <h2>About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p id="ltsconnct">Let's CONNEECT</p>
+                <ul class="img-icons">
+                    <li><a href="#"><img src="img/twitter.jpg" alt="twitter icon"></a></li>
+                    <li><a href="#"><img src="img/Linkedin icon.jpg" alt="Linked in icon"><a></li>
+                    <li><a href="#"><img src="img/medium.jpg" alt="medium icon"></a></li>
+                    <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
+                    <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
+                </ul>
+                <button class="about-btn btn">get my Resume</button>
+            </section>
+                <div class="about-languages">
+                        <img class="iconss" src="img2/angle-down.png" alt="">
+                        <P>Languages</P>
+                </div>
+                <section class="container">
+                    <div class="lang-icon">
+                        <div class="lang">
+                            <img id="im" src="img2/Ellipse 1.png" alt="javascript">
+                            <p>javascript</p>
+                        </div>
+                        <div class="lang">
+                            <img src="img2/Ellipse 2.png" alt="html">
+                            <p>HTML</p>
+                        </div>
+                        <div class="lang">
+                            <img src="img2/Ellipse 3.png" alt="css">
+                            <p>Css</p>
+                        </div>
+                    </div>
+                </section>
```

Comment:
Kindly I suggest you use the `div` tag instead of the `section` tag here to follow HTML semantic and best practices. 😄